home.html

```diff
@@ -7,6 +7,66 @@
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
 </head>
 <body>
+  <header>
+    <!-- Navbar Start -->
+    <nav class="navbar container navbar-expand-lg">
+        <div class="container-fluid px-0">
+            <div class="rectangle-nav"></div>
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                <ul class="navbar-nav ms-auto mb-lg-0">
+                    <li>
+                        <a class="nav-link mx-2" href="#description">Our Services</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link mx-2" href="#why-us">Why Us</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link mx-2" href="#testimonial">Testimonial</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link mx-2" href="#faq">FAQ</a>
+                    </li>
+                    <li class="nav-item">
+                        <button class="btn btn-success mx-2">Register</button>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+    <!-- Navbar End -->
+</header>
+<main>
+  <!-- Hero Start -->
+    <section id="head" class="pt-4 me-0 h-auto">
+        <div class="container">
+            <div class="row">
+                <div class="col">
+                    <div class="d-flex h-100">
+                        <div class="justify-content-center align-self-center">
+                            <h2 class="fw-bold my-3" style="font-size: 36px;">Sewa & Rental Mobil Terbaik di kawasan
+                                Blora</h2>
+                            <p class="fw-lighter my-3" style="font-size: 14px;">Selamat datang di Binar Car Rental.
+                                Kami menyediakan mobil
+                                kualitas terbaik dengan
+                                harga
+                                terjangkau. Selalu siap melayani kebutuhanmu untuk sewa mobil selama 24 jam.</p>
+                            <button class="btn btn-success my-3">Mulai sewa mobil</button>
```

Comment:
change the color of the button to outline primary and change the size to large button